sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -715,14 +718,13 @@ class PlanResolutionSuite extends AnalysisTest {
         val parsed2 = parseAndResolve(sql2)
         val parsed3 = parseAndResolve(sql3)
 
-        val tableIdent = TableIdentifier(tblName, None)
```

Comment:
We only need `TableIdentifier` for v1 command. How about we move this into `if (useV1Command)` and change it to `TableIdentifier(tblName, Some("default"))`?

Reply:
OK. I also reverted some other changes to have minimum diff.